long_term/load_long.py

```diff
@@ -30,6 +30,45 @@ def get_s3_client():
                   aws_secret_access_key=ENV["AWS_rvbyaulf_SECRET_KEY"])
 
 
+def get_ses_clent():
+    """Returns boto3 ses client."""
+
+    return client(service_name="ses", region_name='eu-west-2',
+                  aws_access_key_id=ENV["AWS_rvbyaulf_KEY"],
+                  aws_secret_access_key=ENV["AWS_rvbyaulf_SECRET_KEY"])
+
+
+def send_email(ses_client, to_email: str, from_email: str, df: pd.DataFrame):
+    """Sends an HTML email to the recipient with the DataFrame content."""
+    html_table = df.to_html(index=False)
+
+    html_body = f"""
+    <html>
+    <body>
+        <h2>Archived Plant Readings</h2>
+        <p>Successfully uploaded the following .csv summary to the S3 bucket:</p>
+        {html_table}
+    </body>
+    </html>
+    """
+    ses_client.send_email(
+        Source=from_email,
+        Destination={
+            'ToAddresses': [to_email]
+        },
+        Message={
+            'Subject': {
+                'Data': "Archived Plant Readings to S3"
+            },
+            'Body': {
+                'Html': {
+                    'Data': html_body
+                }
+            }
+        }
+    )
+
+
```

Comment:
Good idea doing this in the code, could this be changed to a step function in the future?

Reply:
Good idea for future improvements!